src/SpineAnimationRenderer.ts

```diff
@@ -153,34 +188,38 @@ export class SpineAnimationRenderer extends Renderer {
    * @internal
    */
   override update(delta: number): void {
-    const { _state, _skeleton } = this;
-    if (_state && _skeleton) {
-      _state.update(delta);
-      _state.apply(_skeleton);
-      _skeleton.update(delta);
-      _skeleton.updateWorldTransform(Physics.update);
-      SpineAnimationRenderer._spineGenerator.buildPrimitive(this._skeleton, this);
-      if (this._isContainDirtyFlag(SpineAnimationUpdateFlags.InitialVolume)) {
-        this._onWorldVolumeChanged();
-        this._setDirtyFlagFalse(SpineAnimationUpdateFlags.InitialVolume);
-      }
-      if (this._isContainDirtyFlag(SpineAnimationUpdateFlags.AnimationVolume)) {
-        this._calculateGeneratorBounds(this.bounds);
-      }
+    const { _state: state, _skeleton: skeleton } = this;
+    if (!state || !skeleton) return;
+    let shouldUpdateBounds = this.updateBoundsPerFrame;
+    if (this._needsInitialize) {
+      this._applyDefaultConfig();
+      shouldUpdateBounds = true;
+      this._needsInitialize = false;
     }
+    state.update(delta);
+    state.apply(skeleton);
+    skeleton.update(delta);
+    skeleton.updateWorldTransform(Physics.update);
+    SpineAnimationRenderer._spineGenerator.buildPrimitive(
+      this._skeleton,
+      this,
+      shouldUpdateBounds
+    );
+    // @ts-ignore
+    shouldUpdateBounds && this._updateBounds(this._bounds);
```

Comment:
Shouldn't depend on update, should get to compute 

You can check `MeshRenderer` _updateBounds logic

Reply:
done